.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,39 @@
+name: Chorus .NET6
+
+on:
+  [push, pull_request, workflow_dispatch]
+
+jobs:
+  build_test_deploy:
+    env:
+      NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    permissions: write-all
+    
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # fetch full history for GitVersion
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+        source-url: https://nuget.pkg.github.com/sillsdev/index.json
+    - name: Fix Python on Linux (for mercurial)
+      if: matrix.os == 'ubuntu-latest'
+      run: sudo apt install python-is-python2
```

Comment:
LibChorus references SIL.Chorus.Mercurial 3.0.3.4 which has the explicit reference to python2, so IMO it would be better to install the python2 package. While it probably doesn't matter for GHA it might break the system if you try to follow these steps on a normal machine where you have other Python apps because it redirects python to python2, but on modern systems python is expected to be python3.

Reply:
per y'all's comments yesterday, i tried both mercurial and python. both of those are already installed. this line was the only that made a difference!